components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/cache/AuthenticationContextCacheEntry.java

```diff
@@ -50,4 +51,14 @@ public String getLoggedInUser() {
     public void setLoggedInUser(String loggedInUser) {
         this.loggedInUser = loggedInUser;
     }
+
+    public long getCreatedTimeNano() {
```

Comment:
Instead of created time, won't it be easy to store the expiry time?

Reply:
I have added created time since we already have the validity period for the cache entry